.github/workflows/33-frontend-pr-mutation-testing.yml

```diff
@@ -67,7 +67,7 @@ jobs:
             xargs -I{} sh -c '[ -f "{}" ] && echo "{}"' |           # check if file exists
             xargs printf '%s,' |                                    # convert to comma separated list
             sed 's/.$//' |                                          # remove trailing comma
-            xargs npx stryker run -m --allowEmpty                   # run stryker on changed files
+            xargs -r npx stryker run --mutate                       # run stryker on changed files
```

Comment:
Could we possibly move this change to a different branch? That way, we can keep the commit history clean. Unless there is a reason this change is tied with the UR change?

Reply:
Seems like this is a commit for #38. Could we rebase and remove this from this pr? Thanks!

Reply:
Yeah. I did exactly same as #38. Otherwise, the frontend mutation testing workflow doesn't work. And I'm not very familiar with git rebase ...

Reply:
Or I can wait for #38 merged into main first, then I'll roll back my commit.

Reply:
In this case, I think it's fine since your code can't work without this change. The best thing to do is wait for #38 to be merged first, and then I can help you rebase your commits. On the other hand, if this is merged first, we can also resolve #38 as a duplicate of this PR. Just make sure you write in your PR description that we also changed the workflows.